packages/wordcount/src/defaultSettings.js

```diff
@@ -1,3 +1,32 @@
+/**
+ * @typedef {import("./index").WordCountStrategy} WordCountStrategy
+ */
+
+/**
+ * @typedef {Object} WordCountSettingsFields
+ * @property {RegExp} HTMLRegExp Regular expression that matches HTML tags
+ * @property {RegExp} HTMLcommentRegExp Regular expression that matches HTML comments
+ * @property {RegExp} spaceRegExp Regular expression that matches spaces in HTML
```

Comment:
It should be that these "columns" are aligned:

```suggestion
 * @property {RegExp} HTMLcommentRegExp Regular expression that matches HTML comments
 * @property {RegExp} spaceRegExp       Regular expression that matches spaces in HTML
```

Reference:

- https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#aligning-comments
- https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#custom-types

Of course, it should account for all other properties as well, where it becomes pretty difficult, at least when also trying to adhere to a particular line length (e.g. 80 characters), and when types can be complex when inlined. Usually I try to balance between: Create `@typedef` with shorter names as appropriate in lieu of inline types like below `Partial<{type: WordCountStrategy, shortcodes: string[]}`, e.g. `Partial<WPTranslationMapping>`, vs. relenting on a hard line length (allow 100 or 120 per line within the DocBlock).

Reply:
Thanks for the reminder, @aduth. An attempt at alignment lives in dc253c424d. To be honest, it's not more readable to me, I will leave it for you to decide how to proceed with the alignment :) I don't have a strong opinion.

P.S. To be honest he time spent on this makes me just use `wp-prettier` and stop caring about “readability” all together.

Reply:
> Thanks for the reminder, @aduth. An attempt at alignment lives in [dc253c4](https://github.com/WordPress/gutenberg/commit/dc253c424de4815bf7d7f50cdfd5974081415842). To be honest, it's not more readable to me, I will leave it for you to decide how to proceed with the alignment :) I don't have a strong opinion.

At least as guidelines currently stand, I think the changes you committed are good and should stay.

The general point is interesting, and it's one I'd be curious to raise for discussion at some point (e.g. [weekly JavaScript chat](https://make.wordpress.org/meetings/)). I'm personally conflicted. I feel there is some benefit to general readability in having the aligned columns. The problems stem largely on the end of _getting them to look that way_, between a combination of problems:

- The effort involved in applying the spacing. 
- The difficulty in choice of line length (phrasing, types choice and externalization, etc)

To the first: I definitely acknowledge that it's a huge pain, at least in VSCode. I recall it being a complete non-issue when I'd used Sublime Text + [DocBlockr](https://github.com/spadgos/sublime-jsdocs), which had very smart auto-indentation associated with the <kbd>Tab</kbd> key. I've yet to find something equivalent for VSCode. I even have a work-in-progress VSCode plugin on my local machine because of this exact problem. To be honest, it feels like something we're just lacking good tooling for, but becomes solved once it exists.

>P.S. To be honest he time spent on this makes me just use wp-prettier and stop caring about “readability” all together.

Well, we do have Prettier, but it does nothing for us for JSDoc 😄